File: app/src/main/java/com/fast_prog/dynate/models/Order.java
package com.fast_prog.dynate.models;

import java.io.Serializable;

/**
 * Created by sarathk on 1/9/17.
 */

public class Order implements Serializable {

    public String tripId;
    public String tripNo;
    public String tripFromAddress;
    public String tripFromLat;
    public String tripFromLng;

    public boolean tripFromSelf;

    public String tripFromName;
    public String tripFromMob;
    public String tripToAddress;
    public String tripToLat;
    public String tripToLng;

    public boolean tripToSelf;

    public String tripToName;
    public String tripToMob;
    public String vehicleModel;
    public String vehicleType;
    public String vehicleImage;
    public String scheduleDate;
    public String scheduleTime;
    public String userName;
    public String userMobile;
    public String tripStatus;
    public String tripFilter;
    public String tripSubject;
    public String tripNotes;
    public String tripDId;
    public String tripDNo;
    public String tripDStatus;
    public String tripDDmId;
    public String tripDRate;
    public String tripDIsNegotiable;
    public String tripDDateTime;
    public String tripDFilterName;
    public String dmName;
    public String dmMobNumber;
    public String distanceKm;
    public String distance;
}

File: app/src/main/java/com/fast_prog/dynate/models/Order.kt
package com.fast_prog.dynate.models

import java.io.Serializable

/**
 * Created by sarathk on 1/9/17.
 */

class Order : Serializable {

    var tripId: String? = null
    var tripNo: String? = null
    var tripFromAddress: String? = null
    var tripFromLat: String? = null
    var tripFromLng: String? = null
    var tripFromSelf: Boolean = false
    var tripFromName: String? = null
    var tripFromMob: String? = null
    var tripToAddress: String? = null
    var tripToLat: String? = null
    var tripToLng: String? = null
    var tripToSelf: Boolean = false
    var tripToName: String? = null
    var tripToMob: String? = null
    var vehicleModel: String? = null
    var vehicleType: String? = null
    var vehicleImage: String? = null
    var scheduleDate: String? = null
    var scheduleTime: String? = null
    var userName: String? = null
    var userMobile: String? = null
    var tripStatus: String? = null
    var tripFilter: String? = null
    var tripSubject: String? = null
    var tripNotes: String? = null
    var tripDId: String? = null
    var tripDNo: String? = null
    var tripDStatus: String? = null
    var tripDDmId: String? = null
    var tripDRate: String? = null
    var tripDIsNegotiable: String? = null
    var tripDDateTime: String? = null
    var tripDFilterName: String? = null
    var dmName: String? = null
    var dmMobNumber: String? = null
    var distanceKm: String? = null
    var distance: String? = null
}
